### PR TITLE
Fixes haunted 8-balls

### LINKED ISSUE #83120: Haunted eightball ghost vote is late and answer is random
## Issue Summary
Haunted eightball ghost vote starts only when the do_after is over and it gives a random answer to the user just like a normal magic eightball.
The vote is triggered once a person starts shaking it, but the window is empty and says something like "No question was asked" and does not change if that person says something, although there is a question that can be seen in the chat announcement and in the vote after do_after. The vote works correctly only after the person finished shaking it, but then it's too late because the ball already gave a random answer

## Round ID:
[228450](https://scrubby.melonmesa.com/round/228450)

## Testmerges:
#83043 
#83045 
#83046

### PR DESCRIPTION
**Pull request by tontyGH**
## About The Pull Request

Haunted 8-balls had the condition for their UI flipped, so it would hide it when it was supposed to show it and vice-versa. This fixes #83120

Also fixed some bad UI work so it's a bit nicer

## Scariest Story (Try Not To Get Scared)

https://github.com/user-attachments/assets/ba5341f1-8025-4931-b3d5-73e0ecf82788

## Changelog

:cl:
fix: Haunted 8-balls are now usable again (FINALLY)
fix: Haunted 8-ball UI now correctly displays your selected answer
qol: Made the haunted 8-ball UI look a pinch better. Just a smidge. 
code: Haunted 8-balls no longer use a duplicate answer list
/:cl:

### CODE REVIEW

**Comment by Jackal-boop** [comment]
for the first time in TG history, will the magic 8 ball work

**Comment by timothymtorres** [comment]
I was expecting the video to be gameplay footage testing out the mechanics. Instead I got a spooky cat jumpscare.

**Comment by Xkallubar** [comment]
no way tiktok brainrot making it's way to the /tg/ github 😭😭😭

File: tgui/packages/tgui/interfaces/EightBallVote.tsx [hunk 1]
```diff
@@ -21,32 +29,48 @@ export function EightBallVote(props) {
...
-        {(shaking && (
-          <NoticeBox>No question is currently being asked.</NoticeBox>
-        )) || <EightBallVoteQuestion />}
+    <Window width={300} height={idealHeight}>
+      <Window.Content pb={'2.5em'}>
+        {(!shaking && (
+          <NoticeBox danger textAlign={'center'}>
+            No question is currently being asked.
+          </NoticeBox>
+        )) || (
```
Comment (jlsnow301) [inline tgui/packages/tgui/interfaces/EightBallVote.tsx:40]: Please use a ternary

**Review by jlsnow301 — CHANGES_REQUESTED** [review]

**Pull request merged**